apps/web/src/studio/LocalStudioPageLayout.tsx

```diff
@@ -0,0 +1,48 @@
+import * as Sentry from '@sentry/react';
+import { Outlet } from 'react-router-dom';
+import styled from '@emotion/styled';
+import { LocalStudioStateProvider } from './LocalStudioStateProvider';
+import { css } from '@novu/novui/css';
+
+const AppShell = styled.div`
+  display: flex;
+  width: 100vw;
+  height: 100vh;
+  min-width: 1024px;
+`;
+
+const ContentShell = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 0%;
+  overflow: hidden; // for appropriate scroll
+`;
+
+export function LocalStudioPageLayout() {
+  return (
+    <Sentry.ErrorBoundary
+      fallback={({ error, eventId }) => (
+        <>
+          Sorry, but something went wrong. <br />
+          Our team has been notified and we are investigating.
+          <br />
+          <code>
+            <small style={{ color: 'lightGrey' }}>
+              Event Id: {eventId}.
+              <br />
+              {error.toString()}
+            </small>
+          </code>
+        </>
+      )}
+    >
+      <LocalStudioStateProvider>
+        <AppShell className={css({ '& *': { colorPalette: 'mode.local' } })}>
+          <ContentShell>
+            <Outlet />
+          </ContentShell>
+        </AppShell>
+      </LocalStudioStateProvider>
+    </Sentry.ErrorBoundary>
+  );
+}
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Use Panda styling exclusively‏